app/containers/TracksContainer/TrackDetailsContainer/index.js

```diff
@@ -37,7 +37,7 @@ export function TrackDetailsContainer({ dispatchRequestTrackDetails, trackDetail
 
   return (
     <Wrapper>
-      <If condition={!singleTrackLoading}>
+      <If condition={trackDetails}>
```

Comment:
!isEmpty(trackDetails) is what we would need to add here.

If you do if(trackDetails) and if your trackDetails was an empty object {} then the if condition will pass which we don't want to. If you do !isEmpty(trackDetails} only if it an non empty trackDetail it will get in

Reply:
Got it.